### ISSUE TITLE
Sometimes suddenly, all the hotkeys stop working? a restart fix it

### ISSUE THREAD

**Issue by tuananh** [koekeishiya/skhd#115]
does anyone find sometimes suddenly, all the hotkeys stop working? a restart fix it.

Not sure if it's skhd or catalina problem? the logs doesn't show much.

**Comment by koekeishiya** [comment]
Sounds a lot like #48 to me, but I could be wrong.

**Comment by tuananh** [comment]
oh yeah the log is like that. looks like it's a duplicate of that

**Comment by rhpittman314** [comment]
I know this was left as open but is pretty much closed. But, given the absence of details, an issue I had was very similar. All of my usual alt key codes had returned, and it seemed like the config file was unable to load. Looking at the log, my only detail was this:

"#2:1 expected decl, modifier or key-literal"

Feeling pretty dumb looking at it now. I thought I tried everything, but when opening it in TextMate instead of TextEdit, I noticed there were four previously unreadable characters, <B5><B5><B5><B5>, on line #2 of my config file. Deleted them, restarted skhd, and everything started working again. Not sure what they were or how they were generated when building, but if it sheds light on any future issues, I'm glad I could help.
